vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,194 @@
+1. Commit any remaining changes.
+1. Run in the terminal the command `git remote add origin git@github.com:inbo/_your_package_name_.git` where you replace `_your_package_name_` with the actual repository name.
+1. Run in the terminal the command `git push -u origin master`.
+  This will run the checklist test on GitHub for the first time.
+  An orange circle, green thick mark or red cross will appear next to the latest commit or PR.
+
+### Finalise online setup
+
+1. [Create](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) a personal access token for GitHub.
+Use the scope `public_repo` at minimum.
+If you want to reuse it later, then make sure to store it securely.
+Anyone with your personal access token can access GitHub on your behalf within the scope you gave the token.
+You can revoke old or compromised tokens.
+1. Go to your package repository on GitHub.
+Choose _Settings_ > _Secrets_.
+Add _New repository secret_ with _name_ `PAT` and copy your personal access token in the _value_.
+Save it by clicking the _Add secret_ button.
+1. Run `rorcid::orcid_auth()` and sign in with your [ORCID iD](https://www.orcid.org) if requested.
+Upon success, the function returns a string starting with `Bearer` followed by an alphanumeric UUID, e.g. dc0a6b6b-b4d4-4276-bc89-78c1e9ede56e.
+Copy the alphanumeric UUID, without the `Bearer` prefix.
+Go back to your package repository on GitHub.
+Add a new repository secret with _name_ `ORCID_TOKEN` and the alphanumeric UUID as _value_.
+1. Go to https://codecov.io and login with your GitHub account.
+Search for your package repository in the INBO organisation.
+Copy the token.
+Again we go back to GitHub and create a new repository secret with _name_ `CODECOV_TOKEN` and the token you copied as _value_.
+1. Wait for all tests to finish (either green thick or red cross).
+1. Go in your package GitHub repository to _Settings_ > _Branches_.
+_Add rule_ for _Branch name pattern_ `master` and select the following items and _Save changes_.
+    - `Require status checks to pass before merging`
+    - `Require branches to be up to date before merging`
+    - _Status checks found in the last week for this repository_
+        - `check package`
+        - `macOS-latest (release)`
+        - `ubuntu-18.04 (oldrel)`
+        - `ubuntu-20.04 (devel)`
+        - `windows-latest (release)`
+
+## Get it working with an existing package or update the checklist
+
+### Prerequistes
+
+1. Make sure the package in using [git](https://inbo.github.io/tutorials/tutorials/git_introduction/) as version control system.
+1. The git status must be clean.
+You'll need to commit all changes first.
+It is OK to have Untracked files.
+1. You need a **public** remote git repository at https://github.com/inbo.
+
+### Local setup
+
+1. Install the checklist package with `remotes::install_github("inbo/checklist")`.
```

Comment:
This is not needed (mentioned) in the `starting an R package from scratch`.  I'm not sure it needs to be mentioned, but I would do it in the same way for the 2 explanations.

Reply:
handled in https://github.com/inbo/checklist/pull/38/commits/ad914e2863e00df8354424d3fff611cdce8202d5